app/core/admin.py

```diff
@@ -2,8 +2,11 @@
 from django.contrib.auth.admin import UserAdmin
 
 from core.models import (CourseDetailHighlight, CourseInformationMapping,
-                         CourseSpotlight, SearchFilter, SearchSortOption,
-                         XDSConfiguration, XDSUIConfiguration, XDSUser)
+                         CourseSpotlight, SearchFilter,
```

Comment:
Dont' forget to run `isort` before you merge to clean up all of the dependency references